addon/components/ember-popper-base.js

```diff
@@ -89,7 +120,7 @@ export default class EmberPopperBase extends Component {
 
   @action
   update() {
```

Comment:
The thought here really being future-proofing. If someone eventually _does_ need `update()`, we can add it later, but if the name is already taken...